pkg/registration/registration_client.go

```diff
@@ -112,6 +112,16 @@ func (rClient *K8sRegistrationClient) GetActionPolicy() []*proto.ActionPolicyDTO
 
 	rClient.addActionPolicy(ab, app, appPolicy)
 
+	// 4. node: move, provision; not resize;
+	node := proto.EntityDTO_VIRTUAL_MACHINE
+	nodePolicy := make(map[proto.ActionItemDTO_ActionType]proto.ActionPolicyDTO_ActionCapability)
+	nodePolicy[proto.ActionItemDTO_MOVE] = supported
```

Comment:
kubeturbo is not handling node move, let's remove this line or change it to notsupported